wsd/COOLWSD.cpp

```diff
@@ -2381,6 +2382,26 @@ void COOLWSD::innerInitialize(Application& self)
     if (getSafeConfig(conf, "storage.wopi.reuse_cookies", reuseCookies))
         LOG_WRN("NOTE: Deprecated config option storage.wopi.reuse_cookies - no longer supported.");
 
+    COOLWSD::WASMState = getConfigValue<bool>(conf, "wasm.enable", false)
```

Comment:
Looks like this won't compile since the variable is declared in ENABLE_DEBUG only (?) - I hate the conditionals - so perhaps better to pull it out of that conditional.